event-schema/event_schema.go

```diff
@@ -234,10 +282,6 @@ func (manager *EventSchemaManagerT) handleEvent(writeKey string, event EventT) {
 	eventIdentifier := ""
 	if eventType == "track" {
 		eventIdentifier, ok = event["event"].(string)
-	} else if eventType == "page" {
-		eventIdentifier, ok = event["name"].(string)
```

Comment:
this is needed for page and screen right? ex: event-models could exist at home-page etc
for identify, alias and group, event models are at source level only

Reply:
every model is linked to a writeKey. Is that what you meant